localstack/services/opensearch/cluster.py

```diff
@@ -248,19 +311,78 @@ def __init__(self, enabled: bool, endpoint: str) -> None:
             self.url = None
 
 
+class EndpointProxy:
+    def __init__(self, base_url: str, forward_url: str, custom_endpoint: str) -> None:
+        super().__init__()
+        self.base_url = base_url
+        self.forward_url = forward_url
+        self.custom_endpoint = custom_endpoint
+        self.routing_rules = None
+
+    def register(self):
+        _url = urlparse(self.base_url)
+        self.routing_rules = register_cluster(
+            host=_url.hostname,
+            path=_url.path,
+            forward_url=self.forward_url,
+            custom_endpoint=self.custom_endpoint,
+        )
+
+    def unregister(self):
+        try:
+            for rule in self.routing_rules:
+                from localstack.services.edge import ROUTER
```

Comment:
As above, this import here doesn't need to be a local one.

Reply:
:+1: